about.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles.css" rel="stylesheet"><title>About The Platform</title>
+        <style>
+            @import url('http://fonts.cdnfonts.com/css/cocogoose');
+        </style>
+        <style>
+            @import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+        </style>
+    </head>
+    <body>
+        <header class="navbar-container">
+            <div class="navbar-cont">
+                <nav class="nav-icon-box">
+                    <ul class="nav-icons">
+                        <li><a href="#"><img src="images/Linkedinicon.png" alt=""></a></li>
+                        <li><a href="#"><img src="images/twitter.png" alt=""></a></li>
+                        <li><a href="#">English</a></li>
+                        <li><a href="#">My Page</a></li>
+                        <li><a href="#">Logout</a></li>
+                    </ul>
+                </nav>
+                <nav class="navbar">
+                    <a href="homepage.html" class="logo"><img src="images/afrologo1.png" alt="The plaform logo"></a>
+                    <button class="close-button">&times;</button>
+                    <ul class="nav-items">
+                        <li class="list-items1"><a href="about.html">About</a></li>
+                        <li class="list-items"><a href="#">Program</a></li>
+                        <li class="list-items"><a href="#">Join</a></li>
+                        <li class="list-items"><a href="#">Sponsor</a></li>
+                        <li class="list-items"><a href="#">News</a></li>
+                        <li class="list-items1 campaign"><a href="#">Cc Campaign</a></li>
+                    </ul>
```

Comment:
- [ ] I do see you are using the same `nav-bar` in the mobile view from the `home page`, but the `nav-bar` in the `about page` should have a home link, not a logo, and the about link should be removed.